packages/indexer/src/index.ts

```diff
@@ -4,13 +4,40 @@ import { Indexer as IndexerRpc } from './indexer.gen'
 
 const fetch = typeof global === 'object' ? global.fetch : window.fetch
 
+export interface SequenceIndexerOpts {
+  JWTAuth?: string
+  ProjectAccessKey?: string
+}
+
 export class SequenceIndexer extends IndexerRpc {
+  public jwtAuth?: string
+  public projectAccessKey?: string
+
   constructor(
     hostname: string,
-    public projectAccessKey?: string,
-    public jwtAuth?: string
+    opts: SequenceIndexerOpts
+  );
+  constructor(
+    hostname: string,
+    projectAccessKey?: string,
+    jwtAuth?: string
```

Comment:
I think this is fine

I'm just thinking - should we call it SecretKey as in the Builder UI? cc @moskalyk